### PR TITLE
Finished all-in-one app script

### PR DESCRIPTION
**Pull request by jan-abel-inwt**
see: https://github.com/Pandora-IsoMemo/all-in-one-apps/issues/1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
...
+`all-in-one-app.sh` - script for working with the pandora-isomemo docker apps
 
-* `docker-script.sh` - script for working with local docker images of the Pandora-IsoMemo Apps (WIP!)
+## Features
+
+- List available Docker images and running containers
+- Pull Docker images
+- Start Docker containers
+- Stop running Docker containers
+
```
Comment (arunge) [inline README.md:11]: We could add a screenshot:
image

but maybe first update the title?

**Review by arunge — CHANGES_REQUESTED** [review]
The script will be very helpful! I have nothing to add, the choices are very clear.

We could only make the documentation a bit clearer, please see my notes!

File: README.md [hunk 2]
```diff
@@ -1,5 +1,27 @@
...
+3. Make the script executable: `chmod +x all-in-one-app.sh`
+4. Run the script with sudo permissions: `sudo ./all-in-one-app.sh`
+
+The script will present a menu with options to list Docker images and running 
+containers, pull images, start containers and stop containers. Follow the 
+prompts to perform the desired actions.
+
+## Requirements
+
+docker & netstat
```
[the remark this answers is not in the document]
Comment (arunge) [reply README.md:27]: And for now, we must add that this is only working for Linux!!

**Comment by jan-abel-inwt** [comment]
I updated the readme and changed the menu title. Have a look

File: README.md [hunk 3]
```diff
@@ -1,5 +1,46 @@
-# Pandora-IsoMemo HelperScripts
+# All-In-One App Script
+
+`all-in-one-app.sh` - script for working with the pandora-isomemo docker apps
+
+**ONLY TESTED ON LINUX**
+
+**SHOULD RUN ON MACOS**
+
+**NO SOLUTION FOR WINDOWS**
```
Comment (arunge) [inline README.md:9]:
```suggestion
**CURRENTLY NO SOLUTION FOR WINDOWS**
```

**Review by arunge — APPROVED** [review]
only a tiny remark below

**Pull request merged**